refinery/ui/source/js/user-file-browser/ctrls/user-file-browser-filters-ctrl.js

```diff
@@ -108,15 +123,15 @@
       };
     }
 
-    var promise = $q.defer();
-    var getUserFiles = userFileBrowserFactory.getUserFiles;
-
-    getUserFiles().then(function (solr) {
-      vm.attributeFilters = userFileBrowserFactory.createFilters(solr.facet_field_counts);
-      promise.resolve();
-    }, function () {
-      $log.error('/files/ request failed');
-      promise.reject();
+   /*
```

Comment:
- Avoid duplicate api call and watch added factory variable, which is updated when files-ctrl calls api for the grid data.